File: 백준/Bronze/4153. 직각삼각형/직각삼각형.java
import java.util.Scanner;

public class Main {
    public static void main(String args[]) {
      int A, B, C, max_val, other1, other2;
    Scanner s = new Scanner(System.in);
    while (true){
        A = s.nextInt();
        B = s.nextInt();
        C = s.nextInt();
        if (A == 0 && B == 0 && C == 0){
            break;
        }
        
        if ((A > B) && (A > C)){
            max_val = A;
            other1 = B;
            other2 = C;
        }else if ((B > A) && (B > C)){
            max_val = B;
            other1 = A;
            other2 = C;
        }else{
            max_val = C;
            other1 = A;
            other2 = B;
        }
        
        if (max_val * max_val == other1 * other1 + other2 * other2){
            System.out.println("right");
        }else{
            System.out.println("wrong");
        }
    }
    }
}

File: 백준/Bronze/4153. 직각삼각형/직각삼각형.py
while True:
    A, B, C = map(int, input().split())
    if A == 0 and B == 0 and C == 0: 
        break
    
    if A > B and A > C:
        max_val = A
        other1 = B
        other2 = C
    elif B > A and B > C:
        max_val = B
        other1 = A
        other2 = C
    else:
        max_val = C
        other1 = A
        other2 = B
    
    if max_val ** 2 == other1 ** 2 + other2 ** 2:
        print("right")
    else:
        print("wrong")
